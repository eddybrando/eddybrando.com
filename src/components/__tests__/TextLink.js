import { shallowMount } from "@vue/test-utils";
import Component from "./../TextLink";

describe("TextLink", () => {
  test("renders correctly", () => {
    const wrapper = shallowMount(Component, {
      stubs: ["font-awesome-icon", "g-link"],
      propsData: { label: "Home", to: "/home" },
    });
    expect(wrapper).toMatchSnapshot();
  });
});
