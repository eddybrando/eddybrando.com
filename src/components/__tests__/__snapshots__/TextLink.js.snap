// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextLink renders correctly 1`] = `
<g-link-stub title="Home" to="/home" class="link">
  Home
  <font-awesome-icon-stub icon="[object Object]" class="arrow"></font-awesome-icon-stub>
</g-link-stub>
`;
